### ISSUE TITLE
Bug: FBX using incorrect animation mTicksPerSecond

### ISSUE THREAD

**Issue by pezcode** [assimp/assimp#4197]
Since https://github.com/assimp/assimp/commit/b3e1ee3ca0d825d384044867fc30cd0bc8417be6 FBXImporter sets an incorrect `aiAnimation::mTicksPerSecond`. As @jeanlemotan pointed out in a comment on the original PR, `mTicksPerSecond` should be `1000.0` after this change, rather than `anim_fps`.

**To Reproduce**

1. Load https://github.com/mosra/magnum-plugins/blob/master/src/MagnumPlugins/AssimpImporter/Test/exported-animation.fbx
2. The reported animation key range after dividing by `mTicksPerSecond` is 0 - 312.5
3. It should be 0 - 7.5

**Comment by philipguin** [comment]
Can confirm this, FBX that used to import with correct durations now imports with extremely long durations.

**Comment by aWilson41** [comment]
Yeah I just upgraded to 5.1.4 today and noticed my fbx animation is running very slow now.

**Comment by kimkulling** [comment]
Loook here: https://github.com/assimp/assimp/issues/4206 SAeems to be the same issue.

**Comment by pezcode** [comment]
> Loook here: #4206 . Seems to be the same issue.

The bug I was seeing was fixed with https://github.com/assimp/assimp/commit/052594568a93de24453c97b30f7e762e25b2bc2b, so most likely identical to https://github.com/assimp/assimp/issues/4221.

I'd close this issue, but @aWilson41 seems to still have similar problems with 5.1.4, which already included that revert commit. So something else is going on here.

**Comment by tellypresence** [comment]
[`exported-animation.fbx`](https://github.com/mosra/magnum-plugins/blob/master/src/MagnumPlugins/AssimpImporter/Test/exported-animation.fbx) seems OK now
<img src="https://github.com/user-attachments/assets/de713191-b0e4-49fa-964a-bfd60fb3ea17" width=180 >
